src/contracts/libraries/GPv2SafeERC20.sol

```diff
@@ -1,20 +1,15 @@
 // SPDX-License-Identifier: LGPL-3.0-or-later
-
-pragma solidity >=0.6.0 <0.8.0;
+pragma solidity ^0.7.6;
 
 import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
 
-/**
- * @title GPv2SafeERC20
- * @author Gnosis Developers
- * @dev Gas-efficient version of Openzeppelin's SafeERC20 contract that does not
- * revert when calling a non-contract.
- */
+/// @title GPv2SafeERC20
+/// @author Gnosis Developers
+/// @dev Gas-efficient version of Openzeppelin's SafeERC20 contract that does
+/// not revert when calling a non-contract.
```

Comment:
Is our safe ERC20 Somehow different than what is already provided by openzeppelin?

Reply:
Yes, it is:
> Gas-efficient version of Openzeppelin's SafeERC20 contract that does not revert when calling a non-contract.

See #264 for discussion around this change.